### ISSUE TITLE
[JOSS] Comments and suggestions on paper

### ISSUE THREAD

**Issue by euronion** [IMMM-SFA/tell#37]
* The model and data used seems to be specific for the USA / the 54 BAs in the CONUS area of the USA. This should be highlighted more strongly in the paper and in the documentation. I also recommend considering adding this limitation (US-only model) prominently to the paper title.

* In the paper it is stated that
  > This model accounts for a wide range of other factors affecting electricity demand, including technology change in the building sector, energy prices, and demand elasticities.
  
  Based on my understanding of the paper and the description in the documentation these capabilities of the model stem from coupling it to other models like GCAM-USA. The capabilities of the model do not seem to be inherent (e.g. detailed technical-bottom-up modelling). I suggest this is made more clear in the paper that to use these capabilities, external assumptions and results have to be provided.

* Public vs open data (Line 47f):
  > tell is based entirely on publicly available data ...
  
    The example data is uploaded to Zenodo under CC-BY-4.0 license. If the data is not only available publicly, but also accessible under an open license, consider emphasising this here (e.g. "tell is based entierly on publicly available, open data ...")

* The software also provides easy access to statistical validation results between the forecasts and observed data (cf. example notebooks). I consider that an aspect worthwhile to be mentioned in the paper
* Consider adding the links in line 83-87 as markup links to the respective site, e.g.  [GitHub repository](https://github.com/IMMM-SFA/tell)

**References**
* A reference to the IM3's future climate and population scenarios (line 67f) should be added
* Your model relies on sklearn. Consider citing the sklearn software package: https://scikit-learn.org/stable/about.html#citing-scikit-learn

**Misc**
* Rewrite "open-source" as "open source"

xref https://github.com/openjournals/joss-reviews/issues/4472

**Comment by mcgrathc** [comment]
Please see this PR for the changes: https://github.com/IMMM-SFA/tell/pull/41

Public vs open data (Line 47f):
tell is based entirely on publicly available data ...
The example data is uploaded to Zenodo under CC-BY-4.0 license. If the data is not only available publicly, but also accessible under an open license, consider emphasising this here (e.g. "tell is based entierly on publicly available, open data ..."): **adopted**

The software also provides easy access to statistical validation results between the forecasts and observed data (cf. example notebooks). I consider that an aspect worthwhile to be mentioned in the paper: **adopted**

Consider adding the links in line 83-87 as markup links to the respective site, e.g. [GitHub repository](https://github.com/IMMM-SFA/tell): **adopted**

References
A reference to the IM3's future climate and population scenarios (line 67f) should be added
Your model relies on sklearn. Consider citing the sklearn software package: https://scikit-learn.org/stable/about.html#citing-scikit-learn: **adopted**

Misc
Rewrite "open-source" as "open source"
: **adopted**

(comments on highlighting CONUS and GCAM coupling to follow)

**Comment by mcgrathc** [comment]
**Below addresses the remaining comments on highlighting CONUS and GCAM coupling:**

**Comment:** “The model and data used seems to be specific for the USA / the 54 BAs in the CONUS 
area of the USA. This should be highlighted more strongly in the paper and in the 
documentation. I also recommend considering adding this limitation (US-only model) 
prominently to the paper title.”
**Response:** We thank the reviewer for raising this point. In response we have updated the title of the
paper to reflect that tell is designed to work in the United States. We also clarified in the 
User Guide that tell was designed to work using data from 54 BAs in the United States 
and in conjunction with the GCAM-USA model, but the general approach could be 
adapted to work in other regions with similar data.

“tell was designed to work using data from 54 BAs the U.S. and in conjunction with the 
GCAM-USA model. Thus, it is not immediately extensible to other countries (e.g., in 
Europe). However, the fundamental modeling approach based on MLP models trained on
historical loads and meteorology data could easily be adapted to work in other regions 
with sufficient data.”

**Comment**: “Based on my understanding of the paper and the description in the documentation 
these capabilities of the model stem from coupling it to other models like GCAM-USA. 
The capabilities of the model do not seem to be inherent (e.g. detailed technical-bottom-
up modelling). I suggest this is made more clear in the paper that to use these 
capabilities, external assumptions and results have to be provided.”
**Response:** We modified this sentence in the paper as follows:
“This model accounts for a wide range of other factors affecting electricity demand, 
including technology change in the building sector, energy prices, and demand 
elasticities, which stems from model coupling with the U.S. version of the Global Change 
Analysis Model (GCAM-USA).”

**Comment:** “I also suggest you highlight the aspect of geographical scope and resolution in the 
beginning of the JOSS paper.”
**Response:** The first sentence in the paper’s Summary section touches on both the geographical 
scope and temporal resolution. Because tell is a multiscale model the geographical 
scope cannot be easily summarize in a single sentence. Full details about the 
geographical scope of the model are provided in the Design and Functionality section.

“The purpose of the Total ELectricity Load (`tell`) model is to generate 21st century 
profiles of hourly electricity load (demand) across the Conterminous United States (CONUS).”
